### PR TITLE
Add script and docs for deploying schema changes.

### PR DESCRIPTION
**Pull request by elliotcm**
Documents https://github.com/alphagov/govuk-puppet/pull/4249

https://trello.com/c/6fnMMZaX/642-1-make-govuk-content-schemas-visible-to-the-publishing-api-in-prod

### CODE REVIEW

File: docs/adding-a-new-format.md [hunk 1]
```diff
@@ -40,3 +40,7 @@ using the rake task in content-store:
 
 If there are examples which are invalid, you may want to change your publisher
 and republish the documents.
+
+## Deploying your changes
+
+Deploy your changes with the [Deploy GOVUK Content Schemas](https://deploy.integration.publishing.service.gov.uk/job/Deploy_GOVUK_Content_Schemas/8/console) Jenkins job.  This makes your changes available to the publishing API for validation and dependency resolution.
```
Comment (tijmenb) [inline docs/adding-a-new-format.md]: This links to the console output.

Comment (elliotcm) [inline docs/adding-a-new-format.md]: Good catch, I've updated it to https://deploy.publishing.service.gov.uk/job/Deploy_GOVUK_Content_Schemas/ which doesn't exist yet but will when the puppet stuff is out.

**Comment by elliotcm** [comment]
The whitehall contract tests seem to have been broken for a while.

**Comment by danielroseman** [comment]
Needs a rebase.

**Comment by elliotcm** [comment]
Weird, it rebased cleanly, merge should've worked.  Anyway, rebased.

**Pull request merged**